offix/src/main/java/org/aerogear/offix/ConflictPojo.kt

```diff
@@ -0,0 +1,31 @@
+package org.aerogear.offix
+
+/* This class is the one to which the json response which comes from the server is matched when a conflict happens.
+ */
+class ConflictPojo(
+    val errors: ArrayList<Errors>
+)
+
+class Errors(
+    val extensions: Extensions
+)
+
+class Extensions(
+    val exception: ExceptionClass
+)
+
+class ExceptionClass(
+    val conflictInfo: ConflictData
+)
+
+class ConflictData(
+    val serverState: ServerState
+)
+
+class ServerState(
```

Comment:
This can be anything, so why don't we let this be Generic in nature and ask the user to provide us with the type of this class?

So, we can probably have a third parameter in our `enqueue` method named `conflictHandler` which can be a generic class that maps to the ServerState key in the json.

What do you think?

Reply:
It is really disappointing to see lack of generic handlers. 
Please use graphback to generate different/more complex backend.

Reply:
Yeah, would be making it generic. Was just trying it initially that it works well.